gems/ibm_cloud/ibm_cloud.gemspec

```diff
@@ -0,0 +1,31 @@
+require_relative 'lib/ibm_cloud/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "ibm_cloud"
+  spec.version       = IbmCloud::VERSION
+  spec.authors       = ["Adam Grare"]
+  spec.email         = ["agrare@redhat.com"]
+
+  spec.summary       = %q{Write a short summary, because RubyGems requires one.}
+  spec.description   = %q{Write a longer description or delete this line.}
```

Comment:
I think there's a way to tell the generator what to use for these values.

Reply:
Added to the `.openapi-config.json`